src/lib/operators/join_hash.cpp

```diff
@@ -119,12 +121,13 @@ struct RadixContainer {
 Build all the hash tables for the partitions of Left. We parallelize this process for all partitions of Left
 */
 template <typename LeftType, typename HashedType>
-std::vector<std::optional<HashTable<HashedType>>> build(const RadixContainer<LeftType>& radix_container) {
+std::vector<std::optional<std::unordered_map<HashedType, boost::variant<RowID, PosList>>>> build(
+    const RadixContainer<LeftType>& radix_container) {
   /*
   NUMA notes:
   The hashtables for each partition P should also reside on the same node as the two vectors leftP and rightP.
   */
-  std::vector<std::optional<HashTable<HashedType>>> hashtables;
+  std::vector<std::optional<std::unordered_map<HashedType, boost::variant<RowID, PosList>>>> hashtables;
```

Comment:
`using` could save us this repetition.

Reply:
Not sure yet how to combine a template type with using. :-/
Let's discuss tomorrow.